libdebugger/.gitignore

```diff
@@ -0,0 +1,4 @@
+/Makefile
+/Makefile
+/Makefile.in
+/Makefile.in
```

Comment:
Is there a specific reason that the entries are double?

Reply:
Blame it on GNU Emacs. I've removed this in 446aaa6

And yes, I agee that  this _is_ better put under doc, so that change has been made as well.